### ISSUE TITLE
Percent sign in Soap message causes error when logs at DEBUG

### ISSUE THREAD

**Issue by blofroth** [citrusframework/citrus#311]
# Steps to reproduce:
* Create a maven project from the archetype:
```
 mvn archetype:generate -Dfilter=com.consol.citrus.mvn:citrus
...
3: remote -> com.consol.citrus.mvn:citrus-quickstart-soap (Citrus quickstart project with SOAP client and server)
Choose a number or apply filter (format: [groupId:]artifactId, case sensitive contains): : 3
...
12: 2.7.2
Choose a number: 12: 12
...
Define value for property 'groupId': com.comsol.citrus.test
Define value for property 'artifactId': soap-format-error
Define value for property 'version' 1.0-SNAPSHOT: :
Define value for property 'package' com.comsol.citrus.test: :
Confirm properties configuration:
groupId: com.comsol.citrus.test
artifactId: soap-format-error
version: 1.0-SNAPSHOT
package: com.comsol.citrus.test
 Y: : Y
```
* Edit `src/test/resources/com/comsol/citrus/test/HelloService_Ok_IT.xml`, change the message to contain a % sign:
`<variable name="message" value="Hello %c !"></variable>`
* Run maven: `mvn clean install`'

# Result:
```
21:36:48,855 ERROR        citrus.Citrus| TEST FAILED HelloService_Ok_IT <com.comsol.citrus.test> Nested exception is:
java.util.IllegalFormatConversionException: c != java.util.ArrayList
        at java.util.Formatter$FormatSpecifier.failConversion(Formatter.java:4302)
        at java.util.Formatter$FormatSpecifier.printCharacter(Formatter.java:2869)
        at java.util.Formatter$FormatSpecifier.print(Formatter.java:2757)
        at java.util.Formatter.format(Formatter.java:2520)
        at java.util.Formatter.format(Formatter.java:2455)
        at java.lang.String.format(String.java:2940)
        at com.consol.citrus.ws.message.SoapMessage.toString(SoapMessage.java:149)
        at com.consol.citrus.ws.client.WebServiceClient.send(WebServiceClient.java:110)
        at com.consol.citrus.actions.SendMessageAction.doExecute(SendMessageAction.java:118)
        at com.consol.citrus.actions.AbstractTestAction.execute(AbstractTestAction.java:42)
        at com.consol.citrus.TestCase.executeAction(TestCase.java:229)
        at com.consol.citrus.TestCase.doExecute(TestCase.java:145)
        at com.consol.citrus.actions.AbstractTestAction.execute(AbstractTestAction.java:42)
        at com.consol.citrus.Citrus.run(Citrus.java:281)
        at com.consol.citrus.testng.AbstractTestNGCitrusTest.run(AbstractTestNGCitrusTest.java:112)

```

# Expected result
Passing test. Or at least no runtime errors.

# Suggestion
In `com.consol.citrus.ws.message.SoapMessage#toString` move `super.toString()` to outside the `String.format()`call.

**Comment by christophd** [comment]
To escape `%`, you will need to double it up: `%%`

**Comment by bergerst** [comment]
Not an acceptable solution, if you use %%, the logging will work, but the SOAP request will also contain %%. There needs to be an actual fix.

**Comment by svettwer** [comment]
Hi!

I'll reopen the issue and investigate it asap.

BR,
Sven

**Comment by svettwer** [comment]
Hi!

I've checked your issue again. The issue was solved with Christophs commit 2c56bb2 which is included in 2.7.3.
So please update your Citrus dependency to a version >= 2.7.3

BR,
Sven
